File: src/main/java/com/mysawit/shipment/MysawitShipmentServiceApplication.java
package com.mysawit.shipment;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cache.annotation.EnableCaching;

@EnableCaching
@SpringBootApplication
public class MysawitShipmentServiceApplication {
    public static void main(String[] args) {
        SpringApplication.run(MysawitShipmentServiceApplication.class, args);
    }
}

File: src/main/kotlin/com/mysawit/shipment/MysawitShipmentServiceApplication.kt
package com.mysawit.shipment

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class MysawitShipmentServiceApplication

fun main(args: Array<String>) {
    val context = runApplication<MysawitShipmentServiceApplication>(*args)
    if (context.environment.getProperty("app.test.close-context", Boolean::class.java, false)) {
        context.close()
    }
}
